<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newwidget</class>
 <widget class="QWidget" name="newwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>360</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(46, 47, 48);
color: rgb(255, 255, 255);color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="main_widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(46, 47, 48);</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>选择一个模板：</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="new_widget" native="true">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QWidget" name="select_widget" native="true">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 1px solid rgb(29, 84, 92);
</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>1</number>
            </property>
            <property name="topMargin">
             <number>1</number>
            </property>
            <property name="rightMargin">
             <number>1</number>
            </property>
            <property name="bottomMargin">
             <number>1</number>
            </property>
            <item>
             <widget class="QLabel" name="show_label">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>20</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(70, 70, 70);
border: none
</string>
              </property>
              <property name="text">
               <string>代码模板</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="tableWidget">
              <property name="showGrid">
               <bool>false</bool>
              </property>
              <attribute name="horizontalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderCascadingSectionResizes">
               <bool>false</bool>
              </attribute>
              <row>
               <property name="text">
                <string>BIN</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>COM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>EXE</string>
               </property>
              </row>
              <column/>
              <item row="0" column="0">
               <property name="text">
                <string>BIN</string>
               </property>
               <property name="flags">
                <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
               </property>
              </item>
              <item row="1" column="0">
               <property name="text">
                <string>COM</string>
               </property>
              </item>
              <item row="2" column="0">
               <property name="text">
                <string>EXE</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="set_widget" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 1px solid rgb(208, 208, 208);
background-color: rgb(43, 45, 48);</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>1</number>
            </property>
            <property name="topMargin">
             <number>1</number>
            </property>
            <property name="rightMargin">
             <number>1</number>
            </property>
            <property name="bottomMargin">
             <number>1</number>
            </property>
            <item>
             <widget class="QWidget" name="file_eidt" native="true">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>60</height>
               </size>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QWidget" name="project_name_edit" native="true">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">border:none
</string>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <property name="spacing">
                   <number>2</number>
                  </property>
                  <property name="leftMargin">
                   <number>10</number>
                  </property>
                  <property name="topMargin">
                   <number>2</number>
                  </property>
                  <property name="rightMargin">
                   <number>10</number>
                  </property>
                  <property name="bottomMargin">
                   <number>2</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="project_label">
                    <property name="minimumSize">
                     <size>
                      <width>30</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>30</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">border:none
</string>
                    </property>
                    <property name="text">
                     <string>项目：</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QWidget" name="project_widget" native="true">
                    <property name="styleSheet">
                     <string notr="true"/>
                    </property>
                    <layout class="QHBoxLayout" name="horizontalLayout_5">
                     <property name="spacing">
                      <number>0</number>
                     </property>
                     <property name="leftMargin">
                      <number>3</number>
                     </property>
                     <property name="topMargin">
                      <number>2</number>
                     </property>
                     <property name="rightMargin">
                      <number>3</number>
                     </property>
                     <property name="bottomMargin">
                      <number>2</number>
                     </property>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QWidget" name="path_edit" native="true">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">border:none
</string>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_4">
                  <property name="spacing">
                   <number>2</number>
                  </property>
                  <property name="leftMargin">
                   <number>10</number>
                  </property>
                  <property name="topMargin">
                   <number>2</number>
                  </property>
                  <property name="rightMargin">
                   <number>10</number>
                  </property>
                  <property name="bottomMargin">
                   <number>2</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="path_label">
                    <property name="minimumSize">
                     <size>
                      <width>30</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>30</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">border:none
</string>
                    </property>
                    <property name="text">
                     <string>路径：</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QWidget" name="path_widget" native="true">
                    <property name="styleSheet">
                     <string notr="true"/>
                    </property>
                    <layout class="QHBoxLayout" name="horizontalLayout_6">
                     <property name="spacing">
                      <number>0</number>
                     </property>
                     <property name="leftMargin">
                      <number>3</number>
                     </property>
                     <property name="topMargin">
                      <number>2</number>
                     </property>
                     <property name="rightMargin">
                      <number>3</number>
                     </property>
                     <property name="bottomMargin">
                      <number>2</number>
                     </property>
                     <item>
                      <widget class="QWidget" name="widget_3" native="true">
                       <layout class="QHBoxLayout" name="horizontalLayout_8">
                        <property name="spacing">
                         <number>0</number>
                        </property>
                        <property name="leftMargin">
                         <number>0</number>
                        </property>
                        <property name="topMargin">
                         <number>0</number>
                        </property>
                        <property name="rightMargin">
                         <number>0</number>
                        </property>
                        <property name="bottomMargin">
                         <number>0</number>
                        </property>
                       </layout>
                      </widget>
                     </item>
                     <item>
                      <widget class="QToolButton" name="pathbutton">
                       <property name="minimumSize">
                        <size>
                         <width>22</width>
                         <height>22</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>22</width>
                         <height>22</height>
                        </size>
                       </property>
                       <property name="cursor">
                        <cursorShape>PointingHandCursor</cursorShape>
                       </property>
                       <property name="styleSheet">
                        <string notr="true">QToolButton::hover{
	background-color: rgb(57, 59, 64);
}

QToolButton {
	border:none;
}
</string>
                       </property>
                       <property name="text">
                        <string/>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_5" native="true">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border:none;</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>10</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="addressPrevLabel">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>8</pointsize>
                  </font>
                 </property>
                 <property name="contextMenuPolicy">
                  <enum>Qt::NoContextMenu</enum>
                 </property>
                 <property name="autoFillBackground">
                  <bool>false</bool>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">border:none;
color: rgb(173, 175, 176);</string>
                 </property>
                 <property name="text">
                  <string>项目将创建于:</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_6" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border:none;
</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_9">
               <property name="leftMargin">
                <number>20</number>
               </property>
               <property name="rightMargin">
                <number>20</number>
               </property>
               <item>
                <widget class="QCheckBox" name="checkBox_2">
                 <property name="styleSheet">
                  <string notr="true">border:none;
color: rgb(173, 175, 176);</string>
                 </property>
                 <property name="text">
                  <string>设置为默认目录</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="checkBox">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="layoutDirection">
                  <enum>Qt::LeftToRight</enum>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">border:none;
color: rgb(173, 175, 176);</string>
                 </property>
                 <property name="text">
                  <string>生成main.asm文件</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_4" native="true">
              <property name="styleSheet">
               <string notr="true">border:none</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="select_button">
           <property name="minimumSize">
            <size>
             <width>55</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>55</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(83, 84, 85);</string>
           </property>
           <property name="text">
            <string>确定</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="cancel_button">
           <property name="minimumSize">
            <size>
             <width>55</width>
             <height>5</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>55</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(83, 84, 85);</string>
           </property>
           <property name="text">
            <string>取消</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
